database/database.py

```diff
@@ -132,8 +132,8 @@ def close_database(db):
     create_tables()
 
     # Add data and alert items for Patient ID '1141'
-    add_all_into_data('1141', oxygen='0.95', bps='120/80', pulse='100')
-    add_alerts_item('1141', 'oxygen', '0.95', '0.96', '0.97')
+    add_all_into_data('1141', oxygen=95, bps=120, pulse=100)
+    add_alerts_item('1141', 'oxygen', 95, 96, 97)

```

Comment:
Looks like you are missing diastolic blood pressure (bpd).

This is what the call should look like:
```
add_all_into_data("some_id", oxygen=50, pulse=100, bps=120, bpd=80)
```